templates/resource.nunjucks

```diff
@@ -59,12 +59,16 @@
 {% if (method.responses | length) != 0 %}
 <br/>
 <p><strong><em><big>Possible Responses</big></em></strong></p>
-<p>
-  {%- set comma = joiner() -%}
-  {%- for response in method.responses -%}
-  {{ comma() }} {{ response.code }}
-  {%- endfor -%}
-</p>
+<dl>
+{% for response in method.responses %}
+  <dt>{{ response.code }}</dt>
+  <dd>
+     {% markdown -%}
+        {{ response.description}}
```

Comment:
I know this is super nitpicky, but there should be a space after description

Reply:
You are right, would have said the same ^^